cafebabel/static/css/styles.css

```diff
@@ -1879,12 +1884,80 @@ textarea#body {
   font-size: 1.3rem;
 }
 
+.home h2 {
+  grid-column: 2 / 6;
+}
+
 .home h2:first-of-type {
   grid-column: 1 / 7;
 }
 
-.home h2:nth-of-type(2) {
-  grid-column: 2 / 6;
+.home a.section-link {
+  position: relative;
+  height: 38vw;
+  max-height: 52rem;
+  background-color: aliceblue;
+  text-align: center;
+  padding-top: 55%;
+  font-family: 'sporting_grotesque', sans-serif;
+  font-size: 5vw;
+  bottom: 0;
+  transition: all 0.3s;
+}
+
+.home a.section-link:hover {
+  bottom: 0.3rem;
+}
+
+.home a.section-link:before {
+  font-family: 'icomoon';
+  position: absolute;
+  top: 43%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 5rem;
+}
+
+.home a.section-link:hover:after {
+  width: 0;
+}
+
+a.section-link.impact {
+  background: url('../img/bg-section-impact.jpg') center;
+}
+
+a.section-link.experience {
+  background: url('../img/bg-section-experience.jpg') center;
+}
+
+a.section-link.experience:before {
+  font-size: 4.5rem;
+}
+
+a.section-link.raw {
+  background: url('../img/bg-section-raw.jpg') center;
+}
+
+a.section-link.creative {
+  background: url('../img/bg-section-creative.jpg') center;
+}
+
+.home a.section-link:first-of-type,
+.home a.section-link:nth-of-type(3) {
+  grid-column: 1 / 4;
+}
+
+.home a.section-link:nth-of-type(2),
+.home a.section-link:nth-of-type(4) {
```

Comment:
and `.home a.section-link:nth-of-type(even)`

Reply:
yes https://github.com/cafebabel/cafebabel.com/pull/245/commits/b2dd784ebc29709e626419ccfd1bb210c61e5b07